packages/insomnia/src/ui/routes/runner.tsx

```diff
@@ -941,7 +942,7 @@ export const runCollectionAction: ActionFunction = async ({ request, params }) =
             const matchId = targetRequest.id === nextRequestIdOrName;
             const matchName = targetRequest.name.trim() === nextRequestIdOrName.trim();
             // find the last request with matched name in case multiple requests with same name in collection runner
-            const matchLastIndex = j === requests.slice().reverse().findIndex(req => req.name.trim() === nextRequestIdOrName.trim());
+            const matchLastIndex = j === _.findLastIndex(requests, req => req.name.trim() === nextRequestIdOrName.trim());
```

Comment:
We aim to avoid the use of lodash for these operations. Was the previous function not working as intended?

Reply:
Yes, the previous function changes the pre previous behavior, the card contains more info, and linter complains of findlastindex. May i know why do we avoid lodash?

Reply:
Some reasons to avoid lodash:
- Most of it is not helpful and replaced by latest JS features
- Increases bundle size and makes us dependent on it. (In our case we include it for the pre/post scripts but shouldn't use it in the project)
- More things we need to learn in order to figure out how something works.

I'd suggest using the platform Array.findLastIndex and use a @ts-expect-error for it.
```suggestion
            // @ts-expect-error - Needs upgrade to latest typescript
            const matchLastIndex = j === requests.findLastIndex(req => req.name.trim() === nextRequestIdOrName.trim());
```

Reply:
https://youmightnotneed.com/lodash

Reply:
I have to say that my thought is a bit different: 🫤 :
- `lodash` is a dependency of [many modules](https://github.com/lodash/lodash/network/dependents), so it seems infeasible to totally exclude it from the bundle as it could be indirect dependency.
- It also take time to learn the builtin approach, but I agree that a small part of lodash functions are not intuitive
- And [this repo](https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore) (18.7 stars) and lodash with 59.7 stars.
But im fine to go with the builtin approach as you mentioned more js features come into play.